packages/api-headless-cms-ddb-es/src/operations/entry/index.ts

```diff
@@ -838,16 +848,18 @@ export const createEntriesStorageOperations = (
         /**
          * We update all ES records with data received.
          */
-        const updatedEntryLevelMetaFields = pickEntryMetaFields(entry, isEntryLevelEntryMetaField);
+        const updatedEntryMetaFields = pickEntryMetaFields(entry, isDeletedEntryMetaField);
```

Comment:
I saw we have above:

 ```ts
const updatedDeletedMetaFields = pickEntryMetaFields(storageEntry, isDeletedEntryMetaField);
``` 

Is this needed? Can we use the above result?

Reply:
Nice one, thank you!